vault-plugins/flant_iam/model/role.go

```diff
@@ -68,6 +78,15 @@ func (t *Role) ObjId() string {
 	return t.Name
 }
 
+func (t *Role) Marshal(_ bool) ([]byte, error) {
+	return jsonutil.EncodeJSON(t)
+}
+
+func (t *Role) Unmarshal(data []byte) error {
+	err := jsonutil.DecodeJSON(data, t)
+	return err
+}
+
```

Comment:
These Marshal & Unmarshal  methods should not be here. @yalosev removed this boilerplate from all models.
I suggest we just remove them, because we no longer need to copy-paste this code at all.

Reply:
fixed